src/Momento.Sdk/Config/Transport/IGrpcConfiguration.cs

```diff
@@ -31,7 +32,7 @@ public interface IGrpcConfiguration
     /// interface for the channel options, which allows modifying specific values but does
     /// not allow the caller to use arbitrary strings to set the channel options.)
     /// </summary>
-    public GrpcChannelOptions GrpcChannelOptions { get; }
+    public GrpcChannelOptions? GrpcChannelOptions { get; }
```

Comment:
Made this nullable so that the `Utils.GrpcChannelOptionsFromGrpcConfig` could do a null check before setting the message send/receive sizes. If set to a `new GrpcChannelOptions()` object before reaching that function, `MaxReceiveMessageSize` will already be set to 4mb and it becomes ambiguous as to whether the user actually wants to set it to 4mb or if it was just the default.

Reply:
the impedance mismatch you are running into here is that in all of our other SDKs we've modeled the GRPC config as a custom object that we define and own, whereas in this SDK we are just using the .NET type directly. 

In this model, we will want to avoid defining any new fields on our own config classes that overlap with the things available on the .NET type, and just use the built-in .NET ones to configure things. This may mean that you don't need a `GrpcChannelOptionsFromGrpcConfig` function at all, but I'll have to look more closely at what other fields you're using from our bespoke Config object to make modifications to the ChannelOptions object.